### PR TITLE
Add in Cramer recs by Ticker

### PR DESCRIPTION
**Pull request by jmaslek**
This was a request from InverseCramer ETF on Twitter.  

I also moved this and the daily recommendations to stocks/ba as this seemed to make more sense.

added tests and .gst

Hugo incoming

### CODE REVIEW

File: gamestonk_terminal/stocks/behavioural_analysis/ba_api.py [hunk 1]
```diff
@@ -66,3 +66,5 @@
 from gamestonk_terminal.stocks.behavioural_analysis.finnhub_view import (
     display_stock_price_headlines_sentiment as snews,
 )
+
+from .cramer_view import display_cramer_daily as cramer
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/behavioural_analysis/ba_api.py:70]: Can we stick with full path here? for consistency

File: gamestonk_terminal/stocks/behavioural_analysis/cramer_view.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+from gamestonk_terminal.stocks.behavioural_analysis import cramer_model
+from gamestonk_terminal.rich_config import console
+from gamestonk_terminal.decorators import log_start_end
+
+logger = logging.getLogger(__name__)
+
+
+@log_start_end(log=logger)
+def display_cramer_daily(inverse: bool = True, export: str = ""):
+    """
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/behavioural_analysis/cramer_view.py:29]: main docstring missing

File: gamestonk_terminal/stocks/behavioural_analysis/cramer_view.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    ----------
+    inverse:bool
+        Flag to include inverse recommendation
+    export:str
+        Format to export data
+    """
+
+    recs = cramer_model.get_cramer_daily(inverse)
+    if recs.empty:
+        console.print("[red]Error getting request.\n[/red]")
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/behavioural_analysis/cramer_view.py:41]: I think we should return here, otherwise there will be an error after

**Review by DidierRLopes — COMMENTED** [review]
Screenshot 2022-03-10 at 00 35 38

When no ticker is provided we should get an error for such.

File: gamestonk_terminal/stocks/behavioural_analysis/ba_api.py (hunk 1 above)
Comment (jmaslek) [reply gamestonk_terminal/stocks/behavioural_analysis/ba_api.py:70]: lol.  just copied it over

Screen Shot 2022-03-10 at 9 50 10 AM

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**